src/components/DownloadImageDialog/ImageForm.tsx

```diff
@@ -466,84 +468,96 @@ export const ImageForm = memo(function ImageForm({
 						/>
 					</>
 				)}
-			</Box>
+			</Stack>
 			<Divider variant="fullWidth" sx={{ my: 3, borderStyle: 'dashed' }} />
-			<Button
-				onClick={() => {
+			<Accordion
+				disableGutters
+				elevation={0}
+				expanded={showAdvancedSettings}
+				onChange={() => {
 					setShowAdvancedSettings(!showAdvancedSettings);
 				}}
-				variant="outlined"
-				sx={{ mb: 2 }}
+				sx={{
+					border: 'none',
+					'&::before': {
+						display: 'none',
+					},
+				}}
 			>
-				{showAdvancedSettings ? <RemoveIcon /> : <AddIcon />} Advanced settings
-			</Button>
-			<Collapse in={showAdvancedSettings} collapsedSize={0}>
-				<Box display="flex" flexDirection="column">
-					<FormControl>
-						<FormLabel htmlFor="poll-interval-label" sx={{ display: 'flex' }}>
-							Check for updates every X minutes{' '}
-							<MUILinkWithTracking
-								href={POLL_INTERVAL_DOCS}
-								sx={{
-									display: 'flex',
-									alignItems: 'center',
-									height: '1.5rem',
+				<AccordionSummary
+					expandIcon={<FontAwesomeIcon icon={faChevronRight} />}
+					sx={{ flexDirection: 'row-reverse', gap: 1 }}
+				>
+					<Typography variant="titleSm">Advanced settings</Typography>
+				</AccordionSummary>
+				<AccordionDetails>
+					<Stack>
+						<FormControl>
+							<FormLabel htmlFor="poll-interval-label" sx={{ display: 'flex' }}>
+								Check for updates every X minutes{' '}
+								<MUILinkWithTracking
+									href={POLL_INTERVAL_DOCS}
+									sx={{
+										display: 'flex',
+										alignItems: 'center',
+										height: '1.5rem',
+									}}
+								>
+									<ArticleIcon sx={{ ml: 1, fontSize: '1.15rem' }} />
+								</MUILinkWithTracking>
+							</FormLabel>
+							<TextField
+								id="poll-interval-label"
+								aria-labelledby="poll-interval-label"
+								value={model.appUpdatePollInterval}
+								slotProps={{
+									htmlInput: {
+										name: 'appUpdatePollInterval',
+										autoComplete: 'appUpdatePollInterval-auto-complete',
+									},
+								}}
+								onChange={(event) => {
+									onChange('appUpdatePollInterval', event.target.value);
 								}}
-							>
-								<ArticleIcon sx={{ ml: 1, fontSize: '1.15rem' }} />
-							</MUILinkWithTracking>
-						</FormLabel>
+							/>
+						</FormControl>
+						<InputLabel htmlFor="provision-key-name" sx={{ my: 2 }}>
```

Comment:
I think we can get rid of `my: 2` (I don't see a reason for it to be there). Also we could probably use the `label` property of the `TextField` component instead (true for other components here as well I think)